runtime/src/nonce_keyed_account.rs

```diff
@@ -1,314 +1,292 @@
 use {
     solana_program_runtime::{ic_msg, invoke_context::InvokeContext},
     solana_sdk::{
-        account::{ReadableAccount, WritableAccount},
```

Comment:
GH is doing a terrible job of diff'n these changes, almost impossible to review well :-(

Reply:
Unfortunately, this is a global problem, and is the same in all the refactoring PRs which replace `KeyedAccount` by `BorrowedAccount`. I started using `BorrowedAccount` everywhere in the runtime long before replacing the `KeyedAccount`s so changing the Trait would cause a lot of code churn elsewhere. Or do you mean to remove the Trait from / change `KeyedAccount` instead?